OpenChange/iCalEvent+MAPIStore.m

```diff
@@ -372,6 +372,7 @@ - (void) updateFromMAPIProperties: (NSDictionary *) properties
   if (value)
     {
       NSArray *recipients;
+      int ccFirstIndex;
```

Comment:
Typo: firstCCIndex.

Furthermore, I'd use `NSUInteger` as Cocoa definition states and along the SOGo code is used:

https://developer.apple.com/library/ios/documentation/Cocoa/Reference/Foundation/Classes/NSArray_Class/index.html#//apple_ref/occ/instp/NSArray/count